modules/extensions-spring-support/Heisenberg_result.xsd

```diff
@@ -0,0 +1,2362 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this?

Reply:
this, kids, is why we don't do `git add .`